frontend/store/collaterals.ts

```diff
@@ -59,6 +67,9 @@ export const mutations = {
     setCollateralStatus(state: State, collateralStatus: CollateralStatus) {
         Vue.set(state.collateralStatusesStorage, collateralStatus.type, collateralStatus);
     },
+    reset(state: State) {
+        Object.assign(state, getInitialState());
```

Comment:
Since getInitialState returns a fresh object, can't you just:
```suggestion
        state = getInitialState();
```
? (here and everywhere else)

Reply:
Yes, but then eslint starts screaming about no unused vars 😅